locale/ja/styles.styl

```diff
@@ -0,0 +1 @@
+@import 'base'
```

Comment:
No, it doesn't so far. Since the every single other locale have the same file even it's one line, I've added it, and it's helpful when we need to tweak the style. I guess that's why they've added it at first. Then let me keep it if it's ok. Thanks :)